backend/capellacollab/projects/capellamodels/routes.py

```diff
@@ -77,6 +78,22 @@ def create_new(
         )
 
 
+@router.patch(
+    "/{model_slug}/description",
```

Comment:
```suggestion
    "/{model_slug}",
```

To be consistent in the routes, I'd drop the description. We're patching the model and the description is passed in the body.